### PR TITLE
Seconds to minutes

### LINKED ISSUE #357: Write a Dart program to convert seconds to minutes
### Description

Write a Dart program to convert seconds to minutes

1 minute = 60 seconds

```
Input  : 300
Output : 5
```
<details>
<summary>How to contribute</summary>

- Save the solution in `convert_seconds_to_minutes.dart` file
- Add `convert_seconds_to_minutes.dart` file in `convert-seconds-to-minutes` folder

</details>

### PR DESCRIPTION
**Pull request by anandfresh**
### Why:

Closes codinasion/codinasion#302

### CODE REVIEW

File: program/convert-seconds-to-minutes/convert_seconds_to_minutes.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import 'dart:io';
+void main(){
+  print("Enter the seconds :");
+  var seconds =int.parse(stdin.readLineSync());
```
Comment (victoriacheng15) [inline program/convert-seconds-to-minutes/convert_seconds_to_minutes.dart]: image

Got this error instead

**Review by victoriacheng15 — CHANGES_REQUESTED** [review]

File: program/convert-seconds-to-minutes/convert_seconds_to_minutes.dart (hunk 1 above)
Comment (anandfresh) [reply program/convert-seconds-to-minutes/convert_seconds_to_minutes.dart]: @victoriacheng15  I used to check it in https://replit.com/ . Its working fine in replit online compiler .
Pls check again in this compiler

Comment (victoriacheng15) [reply program/convert-seconds-to-minutes/convert_seconds_to_minutes.dart]: still got an error with replit

image

Comment (anandfresh) [reply program/convert-seconds-to-minutes/convert_seconds_to_minutes.dart]: @victoriacheng15 can you retest now

**Review by victoriacheng15 — APPROVED** [review]
Thank you for fixing!

**Pull request merged**